catalog/transformer/catalog-transformer-preview/src/main/java/org/codice/ddf/transformer/preview/PreviewMetacardTransformer.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.transformer.preview;
+
+import java.io.Serializable;
+import java.util.Map;
+
+import org.apache.commons.lang.StringUtils;
+import org.apache.tika.io.IOUtils;
+
+import ddf.catalog.data.BinaryContent;
+import ddf.catalog.data.Metacard;
+import ddf.catalog.data.impl.BinaryContentImpl;
+import ddf.catalog.data.types.experimental.Extracted;
+import ddf.catalog.transform.CatalogTransformerException;
+import ddf.catalog.transform.MetacardTransformer;
+
+public class PreviewMetacardTransformer implements MetacardTransformer {
+
+    @Override
+    public BinaryContent transform(Metacard metacard, Map<String, Serializable> arguments)
+            throws CatalogTransformerException {
+        if (metacard == null) {
+            throw new CatalogTransformerException("Cannot transform null metacard.");
+        }
+
+        String preview = "No preview text available.";
+        if (metacard.getAttribute(Extracted.EXTRACTED_TEXT) != null && metacard.getAttribute(
+                Extracted.EXTRACTED_TEXT)
+                .getValue() != null) {
+            preview = metacard.getAttribute(Extracted.EXTRACTED_TEXT)
+                    .getValue()
+                    .toString()
+                    .replaceAll("\n", "</br>\n");
```

Comment:
What about \r? Also, it should be `<br />` if you're using XHTML or just `<br>` for HTML and then you don't need to reapply the \n